New "Mugger" Spaceport mission (#6650)
endless-sky/endless-sky: data/human/human missions.txt

	to offer
		has "Transport Workers B: done"
	on offer
		event "fumbling mugger timer" 60
		fail

event "fumbling mugger timer"

mission "Fumbling Mugger"
	name "Transport to <planet>"
	description "Bring a young woman who threatened you on <origin> to <destination>."
	minor
	source
		attributes "dirt belt" "urban"
	destination "Maker"
	passengers 1
	to offer
		random < 45
		has "event: fumbling mugger timer"
	on offer
		conversation
			`Your exploration of the spaceport leads you to a dark alley where the number of working lights are far outnumbered by the number of broken ones. The contents of steel bins lining the walls overflow onto the pavement, attracting swarms of flies to them. On the surfaces that are illuminated, you can see a mess of black scrawlings upon bare, cracked concrete, and the few windows present on the towering walls surrounding you are either boarded up or shattered. A dark crimson stains the bottom half of a broken window.`
			`	Suddenly, you feel a cold object pushed against your back as a woman speaks softly. "Hands where I can see them, Captain."`
			choice
				`	(Lift your hands up and slowly turn.)`
					goto peaceful
				`	(Spin around and fire.)`

> 				`	(Backwards somersault over your attacker and disarm her.)`

Reviewer: I might rewrite this in a slightly more plausible way. :) Like this:

```suggestion
				`	(Try a fancy acrobatic move and disarm your attacker.)`
```

					goto flip
			
			`	You turn around as swiftly as you can as your hand grasps for your pistol. Due to surprise, or perhaps nervousness, the woman threatening you fails to fire the shotgun she's holding in the second you spend turning. You manage to land a shot, and your quarry falls while a cry escapes from her body. As you walk away, her body fades into the darkness, blending with the garbage bags around it.`
				decline
			
			label flip

> 			`	You push down as hard as you can with your legs while leaning backwards, rising to the lofty height of two feet above the ground. As you begin to fall back-first uncontrollably, you feel metal dig into your spine as a gasp escapes from behind you, before landing on a soft body. You pick yourself up as fast as possible, and you end up face-to-face with your attacker. In the darkness, you pick out that she's wearing a black hoodie with a bandanna over her mouth, her back hunched over and feet placed apart. As you look down, you spot a sawed-off break-action shotgun held single-handed, the end of which is pointed at your abdomen. "Hands up now, Captain."`

Reviewer 2: ```suggestion
			`	You push down as hard as you can with your legs while leaning backwards, rising to the lofty height of two feet above the ground. As you begin to fall back-first uncontrollably, cold metal digs into your spine. A gasp escapes from behind as you land on a soft body. You pick yourself up as fast as possible, and you end up face-to-face with your attacker. In the darkness, you pick out that she's wearing a black hoodie with a bandanna over her mouth, her back hunched over and feet placed apart. As you look down, you spot a sawed-off break-action shotgun held single-handed, the end of which is pointed at your abdomen. "Hands up now, Captain."`
```
That `two feet` bit made me chuckle :)
I feel like that middle sentence could use some work - feels very long and messy as-is. If you've got any alternative ideas than the ones I'm suggesting, be my guest!

> 			`	You push down as hard as you can with your legs while leaning backwards, rising to the lofty height of two feet above the ground. As you begin to fall back-first uncontrollably, you feel metal dig into your spine as a gasp escapes from behind you, before landing on a soft body. You pick yourself up as fast as possible, and you end up face-to-face with your attacker. In the darkness, you pick out that she's wearing a black hoodie with a bandanna over her mouth, her back hunched over and feet placed apart. As you look down, you spot a sawed-off breach-action shotgun held single-handed, the end of which is pointed at your abdomen. "Hands up now, Captain."`

Reviewer: ```suggestion
			`	You push down as hard as you can with your legs while leaning backwards, rising to the lofty height of two feet above the ground. As you begin to fall back-first uncontrollably, you feel metal dig into your spine as a gasp escapes from behind you, before landing on a soft body. You pick yourself up as fast as possible, and you end up face-to-face with your attacker. In the darkness, you pick out that she's wearing a black hoodie with a bandanna over her mouth, her back hunched over and feet placed apart. As you look down, you spot a sawed-off break-action shotgun held single-handed, the end of which is pointed at your abdomen. "Hands up now, Captain."`
```

(there's no such thing as a "breach-action shotgun")

			choice
				`	(Put your hands up.)`
					goto handsup
				`	(Draw your gun and fire.)`

> 			`	You hand darts for your gun, and you manage to aim and fire before the woman does. She collapses on the ground while a cry escapes from her body. As you walk away, her body fades into the darkness, blending with the garbage bags around it.`

Reviewer 2: ```suggestion
			`	You hand darts for your gun, and you somehow manage to aim and fire before the woman does. She collapses on the ground while a cry escapes from her body. As you walk away, her body fades into the darkness, blending with the garbage bags around it.`
```

				decline
			
			label peaceful

> 			`	You raise your arms and turn to face the woman. In the darkness, you pick out that she's wearing a black hoodie with a bandanna over her mouth, her back hunched over and feet placed apart. As you look down, you spot a sawed-off breach-action shotgun held single-handed, the end of which is pointed at your abdomen.`

Reviewer: ```suggestion
			`	You raise your arms and turn to face the woman. In the darkness, you pick out that she's wearing a black hoodie with a bandanna over her mouth, her back hunched over and feet placed apart. As you look down, you spot a sawed-off break-action shotgun held single-handed, the end of which is pointed at your abdomen.`
```

			choice
				`	"And if I don't take you there?"`
				`	"Are you going to pay me?"`
					goto payment
				`	"Why do you want to go there?"`
					goto why
			
			`	She scoffs. "What do you think? How else might an encounter involving a gun pointed-"`
				goto fail
			label payment
			`	She scoffs. "Payment? If I wanted to pay you, I wouldn't be in this alley pointing-"`
				goto fail
			label why
			`	She scoffs. "All you need to know is that I've got a gun pointed-"`
			
			label fail
			`	The attacker's speech is interrupted by a click as her shotgun swings open. She takes a step back as her eyes widen with panic.`
			label choice
			choice
				`	(Threaten her with your gun.)`
					goto threaten
				`	(Shoot her.)`
			
			`	Before she can move a muscle, you whip out your pistol and fire, watching as the woman collapses onto the ground with a cry. As you walk away, her body fades into the darkness, blending with the garbage bags around it.`
				decline
			
			label threaten
			`	You take out your gun and aim it at the woman's head. "Drop the gun," you say, "or I'll shoot."`

> 			`	She seizes up for a few moments, before throwing her shotgun away and raising her hands. "I'm, um, sorry about threatening you. Would you mind if the both of us just walked this off?"`

Reviewer 2: ```suggestion
			`	She seizes up for a few moments, then she throws her shotgun away and raises her hands. "I'm, um, sorry about threatening you. Would you mind if the both of us just walked this off?"`
```

			choice
				`	"You don't need to pay me."`
					goto accept
				`	"You can pay me later."`
				`	"Actually, I can't take you there without payment."`
					goto decline
			action
				set "Fumbling Mugger: future payment"
			label accept
			`	The woman's eyes light up. "I can't thank you enough for this." The two of you head out of the alleys and back to the landing pads, and you show her to an empty bunk on your ship.`
				accept
			
			label decline
			`	"I understand." She shuffles away with her head hung, and you head back to a brighter area of the spaceport.`
				decline
	
	on abort
		conversation
			`As you tell the woman about the change in travel plans, she sighs with resignation. "I suppose that I shouldn't expect anything after how I treated you." You recall that she doesn't even have a single credit to her name, and without any money, it is unlikely that she will survive for long.`
			branch poor
				"credits" < 15000
			choice
				`	(Wish her good luck.)`
					goto leave
				`	(Give her 15,000 credits before dropping her off.)`
					goto gift
			label poor
			choice
				`	(Wish her good luck.)`
			label leave

> 			`	She nods in response. "I'll try my best."`

Reviewer 3: ```suggestion
			`	She nods in response. "I'll try my best," she says solemnly as she prepares to depart your ship.`
```

				decline
			label gift
			action
				payment -15000
				set "gifted money to mugger"
				log `Was held up by a woman who wanted transport to Maker. Managed to gain the upper hand in the engagement, but decided to pay for her transport anyway.`
			`	She is almost incredulous at your gift. "I really can't thank you enough for helping me out." The two of you say your farewells as she gets ready to depart your ship.`

> 				decline

Reviewer 3: Don't need an endpoint at the end of a conversation that isn't in an on offer.
```suggestion
```

		fail

event "fumbling mugger timer"

mission "Fumbling Mugger"
	name "Transport to <planet>"
	description "Bring a young woman who threatened you on <origin> to <destination>."
	minor
	source
		attributes "dirt belt" "urban"
	destination "Maker"
	passengers 1
	to offer
		random < 45
		has "event: fumbling mugger timer"
	on offer
		conversation
			`Your exploration of the spaceport leads you to a dark alley where the number of working lights are far outnumbered by the number of broken ones. The contents of steel bins lining the walls overflow onto the pavement, attracting swarms of flies to them. On the surfaces that are illuminated, you can see a mess of black scrawlings upon bare, cracked concrete, and the few windows present on the towering walls surrounding you are either boarded up or shattered. A dark crimson stains the bottom half of a broken window.`
			`	Suddenly, you feel a cold object pushed against your back as a woman speaks softly. "Hands where I can see them, Captain."`
			choice
				`	(Lift your hands up and slowly turn.)`
					goto peaceful
				`	(Spin around and fire.)`
				`	(Try a fancy acrobatic move and disarm your attacker.)`
					goto flip
			

> 			`	You turn around as swiftly as you can as your hand grasps for your pistol. Due to surprise, or perhaps nervousness, the woman threatening you fails to fire the shotgun she's holding in the second you spend turning. You manage to land a shot, and your quarry falls while a cry escapes from her body. As you walk away, her body fades into the darkness, blending with the garbage bags around it.`

Reviewer 3: ```suggestion
			action
				set "fumbling mugger shot"
			`	You turn around as swiftly as you can as your hand grasps for your pistol. Due to surprise, or perhaps nervousness, the woman threatening you fails to fire the shotgun she's holding in the second you spend turning. You manage to land a shot, and your quarry falls while a cry escapes from her body. As you walk away, her body fades into the darkness, blending with the garbage bags around it.`
```

				decline
			
			label flip
			`	You push down as hard as you can with your legs while leaning backwards, rising to the lofty height of one foot above the ground. As you begin to fall back-first uncontrollably, cold metal digs into your spine. A gasp escapes from behind as you land on a soft body. You pick yourself up as fast as possible, and you end up face-to-face with your attacker. In the darkness, you pick out that she's wearing a black hoodie with a bandanna over her mouth, her back hunched over and feet placed apart. As you look down, you spot a sawed-off break-action shotgun held single-handed, the end of which is pointed at your abdomen. "Hands up now, Captain."`
			choice
				`	(Put your hands up.)`
					goto handsup
				`	(Draw your gun and fire.)`

> 			`	You hand darts for your gun, and you somehow manage to aim and fire before the woman does. She collapses on the ground while a cry escapes from her body. As you walk away, her body fades into the darkness, blending with the garbage bags around it.`

Reviewer 3: ```suggestion
			action
				set "fumbling mugger shot"
			`	You hand darts for your gun, and you somehow manage to aim and fire before the woman does. She collapses on the ground while a cry escapes from her body. As you walk away, her body fades into the darkness, blending with the garbage bags around it.`
```

				decline
			
			label flip

> 			`	You push down as hard as you can with your legs while leaning backwards, rising to the lofty height of one foot above the ground. As you begin to fall back-first uncontrollably, cold metal digs into your spine. A gasp escapes from behind as you land on a soft body. You pick yourself up as fast as possible, and you end up face-to-face with your attacker. In the darkness, you pick out that she's wearing a black hoodie with a bandanna over her mouth, her back hunched over and feet placed apart. As you look down, you spot a sawed-off break-action shotgun held single-handed, the end of which is pointed at your abdomen. "Hands up now, Captain."`

Reviewer 3: I feel like there's something more comedic about two feet than one foot, so I'd flip it back.
